pkg-manager/plugin-commands-installation/test/install.ts

```diff
@@ -52,3 +53,126 @@ test('install with no store integrity validation', async () => {
 
   expect(fs.readFileSync('node_modules/is-positive/readme.md', 'utf8')).toBe('modified')
 })
+
+describe('install injected dependencies', () => {
```

Comment:
this description doesn't explain what the test is actually about. Maybe mention that it is about the manifest of the injected dependencies.

Reply:
Took a swing at updating to mention manifest + publishConfig